src/components/HeaderDropdown/index.tsx

```diff
@@ -6,13 +6,13 @@ const styles = require('./index.less')
 
 interface HeaderDropdownProps {
   children: Object
-  overlayClassName: Object
-  placement: string
+  overlay: Object
+  placement?: 'topLeft' | 'topCenter' | 'topRight' | 'bottomLeft' | 'bottomCenter' | 'bottomRight'
 }
 
 export default function HeaderDropdown (props: HeaderDropdownProps) {
-  const { overlayClassName } = props
+  const { overlay } = props
```

Comment:
:+1: 

But maybe we can use `props` directly - than the function can be almost one liner ;)

```
return <Dropdown overlay={classNames(styles.container, props.overlay)} {...props} />
```

Reply:
updated